DCE Issue with SIMD functiions

I get errors like these when building:

```
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv4_8_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv8_8_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv4_10_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv8_10_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv4_12_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv8_12_sse4 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_h32_8_avx2 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_h16_10_avx2 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_v32_8_avx2 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_v16_10_avx2 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_qpel_hv16_10_avx2 referenced in function ff_hevc_dsp_init_x86
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w4_8_sse4 referenced in function ff_hevc_put_hevc_uni_w_epel_h4_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w4_8_sse4 referenced in function ff_hevc_put_hevc_bi_w_epel_h4_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w6_8_sse4 referenced in function ff_hevc_put_hevc_uni_w12_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w6_8_sse4 referenced in function ff_hevc_put_hevc_bi_w12_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w8_8_sse4 referenced in function ff_hevc_put_hevc_uni_w16_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w8_8_sse4 referenced in function ff_hevc_put_hevc_bi_w16_8_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w4_10_sse4 referenced in function ff_hevc_put_hevc_uni_w_epel_h4_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w4_10_sse4 referenced in function ff_hevc_put_hevc_bi_w_epel_h4_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w6_10_sse4 referenced in function ff_hevc_put_hevc_uni_w12_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w6_10_sse4 referenced in function ff_hevc_put_hevc_bi_w12_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w8_10_sse4 referenced in function ff_hevc_put_hevc_uni_w16_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w8_10_sse4 referenced in function ff_hevc_put_hevc_bi_w16_10_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w4_12_sse4 referenced in function ff_hevc_put_hevc_uni_w_epel_h4_12_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w4_12_sse4 referenced in function ff_hevc_put_hevc_bi_w_epel_h4_12_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w6_12_sse4 referenced in function ff_hevc_put_hevc_uni_w12_12_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_bi_w6_12_sse4 referenced in function ff_hevc_put_hevc_bi_w12_12_sse4
11>libavcodec.lib(libavcodec_x86_hevc_dsp_init.obj) : error LNK2019: unresolved external symbol ff_hevc_put_hevc_uni_w8_12_sse4 referenced in function ff_hevc_put_hevc_uni_w16_12_sse4
11>libav
```

--- a/source/projectGenerator.cpp
+++ b/source/projectGenerator.cpp
@@ -1001,11 +1001,11 @@ void ProjectGenerator::outputSourceFiles(string& projectTemplate, string& filter
     outputSourceFileType(m_includesRC, "ResourceCompile", "Resource", projectTemplate, filterTemplate, foundObjects,
         foundFilters, false, false, true);
 
-    // Output ASM files in specific item group (must go first as asm does not allow for custom obj filename)
+    // Output ASM files in specific item group
     if (!m_includesASM.empty()) {
         if (m_configHelper.isASMEnabled()) {
             outputSourceFileType(m_includesASM, (m_configHelper.m_useNASM) ? "NASM" : "YASM", "Source", projectTemplate,
-                filterTemplate, foundObjects, foundFilters, false);
+                filterTemplate, foundObjects, foundFilters, true);
         } else {
             outputError("Assembly files found in project but assembly is disabled");
         }

